app/value_objects/attachment.rb

```diff
@@ -21,4 +21,8 @@ def filename_with_extension
 
     "#{raw_filename}.#{ext}"
   end
+
+  def size
+    File.size(path)
+  end
```

Comment:
Do you think is worth to have a single test for this method?

Reply:
yeah i wondered about that, but then decided not to as it would just be testing the ruby File class itself. 

but, i considered a test around what happens when the temp file hasn't been written and the size errors because it can't file the file. there is currently nothing regarding that anywhere. worth chatting about.